### ISSUE TITLE
Fix protocol loading (Trac #55)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#59]
Somewhere prior to version 0.1.325, protocol loading was broken. There are no errors, but the protocol doesn't change after loading a file.

Migrated from http://microfluidics.utoronto.ca/ticket/55
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "Somewhere prior to version 0.1.325, protocol loading was broken. There are no errors, but the protocol doesn't change after loading a file.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Fix protocol loading", 
    "priority": "critical", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-02-20T04:20:27", 
    "milestone": "Microdrop 1.0", 
    "owner": "ryan", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120220T04:28:41: ryan changed priority from "major" to "critical"

**Comment by cfobel** [comment]
Trac update at 20120221T18:16:00: 
* ryan@fobel.net commented:

> commit e75809f0c114d9ddc9a19aa49fcb2d6e005f8dd5
> Author: Ryan Fobel <ryan@fobel.net>
> Date:   Tue Feb 21 12:53:30 2012 -0500
> 
>     [SUBMODULE] change the way that dmf_device/protocol/experiment_log are updated
>     
>     Fixes #59.
>     
>     This undoes some of the changes introduced in 15acb2d and makes the App object
>     responsible for updating its own references to the DmfDevice, Protocol and
>     ExperimentLog objects.
>     
>     This commit also makes the ExperimentLog class update it's experiment id on
>     creation.
>     
>     Finally, this commit removes the MainWindowController's update function, instead
>     adding handlers to watch for on_dmf_device_changed, on_protocol_changed and
>     on_experiment_log_changed.


* ryan@fobel.net changed resolution from "" to "fixed"
* ryan@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
